Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -3502,10 +3503,9 @@ class CloneDeserializer : public CloneBase {
             if (span.size() < length)
                 return false;
```

Comment:
Could use `consumeSpan` to do this length check to and put the `span` into a local variable.

Reply:
`consumeSpan()`, as proposed, doesn't do a length check (it would RELEASE_ASSERT). I guess we could make it do a length check and return an empty span, then leave it to the caller to check that they got a span for the right size?